CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ### New features
 
+#### We've adjusted our responsive type scale
+
+We've made the following adjustments to our responsive type scale:
+
+- point 16 is now 16px across all screen sizes
+- point 19 is now 19px across all screen sizes
+- point 24 is still 24px on large screens, but on small screens is now font size 21px instead of 18px and line height 25px instead of 20px
+- point 27 is still 27px on large screens, but on small screens is now font size 21px instead of 18px and line height 25px instead of 20px
```

Comment:
```suggestion
- 27px remains the same on large screens
- 27px is now 21px on small screens instead of 18px and has a line height 25px instead of 20px
```